provider/lxd/environ_instance.go

```diff
@@ -123,3 +123,22 @@ func (env *environ) parsePlacement(placement string) (*instPlacement, error) {
 
 	return nil, errors.Errorf("unknown placement directive: %v", placement)
 }
+
+// MoveInstancesToController updates the controller tags on the
```

Comment:
MoveInstancesToController is a bit of a mouthful. How about "AdoptInstances", to keep it brief?